pkg/plugins/plugins.go

```diff
@@ -148,13 +148,17 @@ func Get(name, imp string) (*Plugin, error) {
 	if err != nil {
 		return nil, err
 	}
-	for _, driver := range pl.Manifest.Implements {
-		logrus.Debugf("%s implements: %s", name, driver)
-		if driver == imp {
-			return pl, nil
+	if imp != "" {
```

Comment:
Just realizing why `if imp != "" {` is necessary here.  Since this is a shared package between volume/network, we want to delay the implement lookup.  So in the case of the Volume Driver, we always skip passing the Impl which bypasses this lookup.  Whereas, the networking will continue functioning as normal.